### PR TITLE
Improve report

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: src/main/java/org/sonarlint/cli/report/source/HtmlSourceDecorator.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.cli.report.source;
+
+import java.util.List;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+import org.sonarlint.cli.report.IssueWithId;
+
+public class HtmlSourceDecorator {
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/cli/report/source/HtmlSourceDecorator.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1118)

File: src/main/java/org/sonarlint/cli/report/HtmlReport.java [hunk 2]
```diff
@@ -77,13 +79,11 @@ public void print(IssuesReport report) {
 
   private void writeToFile(IssuesReport report, Path toFile) {
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/cli/report/HtmlReport.java]: MAJOR Make "writeToFile" a "static" method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2325)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM

**Pull request merged**